app-backend/db/src/main/scala/ProjectDao.scala

```diff
@@ -76,6 +75,8 @@ object ProjectDao
 
   def insertProject(newProject: Project.Create,
                     user: User): ConnectionIO[Project] = {
+    val s: Scene = ???
+    s.filterFields.acquisitionDate.get.toLocalDateTime.get(IsoFields.WEEK_OF_WEEK_BASED_YEAR)
```

Comment:
Not sure I get what's going on here.

Reply:
Oh oops, from a debugging session - I'm gonna remove this

Reply:
Resolved in https://github.com/raster-foundry/raster-foundry/pull/4733/commits/32aea7b0aa17f69614ec7a1feb2cb5480d58f712